parsec/api/protocol/cmds.py

```diff
@@ -104,7 +106,7 @@
 # TODO: replace administration cmds by a dedicated HTTP REST API
 APIV1_ADMINISTRATION_CMDS = {
     "organization_create",
-    "organization_stats",
+    "apiv1_organization_stats",
```

Comment:
Renaming the command breaks api backward compatibility !

Reply:
fixed